### DISCUSSION TITLE
Multimodal

### DISCUSSION THREAD

**Discussion by SorinHornoiu** [FlowiseAI/Flowise#1831] [General]
Hi, I just saw that in the last version (1.6.0) there is a new capability, “Multimodal”. How could be used? Thank you

**Comment by vinodkiran** [comment]
It is currently available for ChatOpenAI (gpt-vision). Build a flow with ChatOpenAI model and enable the "allow images upload". Use the LLM Chain or Conversation Chain. 

Save the chat flow. 

The Chat window should show an image button to the left of the text input, allowing you to upload an image and ask question.

**Comment by Spider201866** [comment]
Option to upload is available directly in the chat window, but not when using the share chatbot page?

Separately, and as many always ask, it'd be nice to have more options on this page anyway: font sizes, bolding, rounding of text box corners, colours etc, drag n drop of avatars. All for speedy and easy work...
